merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/remotes/postgres/InsertSimulationEventsAction.java

```diff
@@ -31,16 +32,15 @@ public InsertSimulationEventsAction(final Connection connection) throws SQLExcep
 
   public void apply(
       final long datasetId,
-      final Map<Duration, List<EventGraph<Pair<Integer, SerializedValue>>>> eventPoints,
-      final Timestamp simulationStart
+      final Map<Duration, List<EventGraph<Pair<Integer, SerializedValue>>>> eventPoints
   ) throws SQLException {
     for (final var eventPoint : eventPoints.entrySet()) {
       final var time = eventPoint.getKey();
       final var transactions = eventPoint.getValue();
       for (int transactionIndex = 0; transactionIndex < transactions.size(); transactionIndex++) {
         final var eventGraph = transactions.get(transactionIndex);
         final var flattenedEventGraph = EventGraphFlattener.flatten(eventGraph);
-        batchInsertEventGraph(datasetId, time, transactionIndex, simulationStart, flattenedEventGraph, this.statement);
+        batchInsertEventGraph(datasetId, time, transactionIndex, flattenedEventGraph, this.statement);
```

Comment:
Why was this done via a static method rather than inlined in the `apply` like our other batch inserts?